[2024-07-01T12:25:45.982+0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-01T12:25:46.105+0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Ingest_Data.wait_for_fetch_task scheduled__2024-06-30T00:00:00+00:00 [queued]>
[2024-07-01T12:25:46.111+0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Ingest_Data.wait_for_fetch_task scheduled__2024-06-30T00:00:00+00:00 [queued]>
[2024-07-01T12:25:46.122+0300] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-01T12:25:46.141+0300] {taskinstance.py:2330} INFO - Executing <Task(ExternalTaskSensor): wait_for_fetch_task> on 2024-06-30 00:00:00+00:00
[2024-07-01T12:25:46.178+0300] {standard_task_runner.py:63} INFO - Started process 2154 to run task
[2024-07-01T12:25:46.188+0300] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Ingest_Data', 'wait_for_fetch_task', 'scheduled__2024-06-30T00:00:00+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/ingest_DAG.py', '--cfg-path', '/tmp/tmpydb2kzhb']
[2024-07-01T12:25:46.192+0300] {standard_task_runner.py:91} INFO - Job 66: Subtask wait_for_fetch_task
[2024-07-01T12:25:46.255+0300] {task_command.py:426} INFO - Running <TaskInstance: Ingest_Data.wait_for_fetch_task scheduled__2024-06-30T00:00:00+00:00 [running]> on host DESKTOP-1R67MDQ.
[2024-07-01T12:25:46.324+0300] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Ingest_Data' AIRFLOW_CTX_TASK_ID='wait_for_fetch_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-30T00:00:00+00:00'
[2024-07-01T12:25:46.325+0300] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-01T12:25:46.345+0300] {baseoperator.py:399} WARNING - ExternalTaskSensor.execute cannot be called outside TaskInstance!
[2024-07-01T12:25:46.345+0300] {external_task.py:258} INFO - Poking for tasks ['end_task'] in dag fetch on 2024-06-30T00:00:00+00:00 ... 
[2024-07-01T12:25:46.352+0300] {logging_mixin.py:188} WARNING - /home/user/rubix/internship_project/.venv/lib/python3.10/site-packages/airflow/utils/session.py:79 DeprecationWarning: This method is deprecated and will be removed in future.
[2024-07-01T12:26:46.358+0300] {external_task.py:258} INFO - Poking for tasks ['end_task'] in dag fetch on 2024-06-30T00:00:00+00:00 ... 
[2024-07-01T12:27:46.373+0300] {external_task.py:258} INFO - Poking for tasks ['end_task'] in dag fetch on 2024-06-30T00:00:00+00:00 ... 
[2024-07-01T12:28:46.390+0300] {external_task.py:258} INFO - Poking for tasks ['end_task'] in dag fetch on 2024-06-30T00:00:00+00:00 ... 
[2024-07-01T12:29:46.438+0300] {external_task.py:258} INFO - Poking for tasks ['end_task'] in dag fetch on 2024-06-30T00:00:00+00:00 ... 
[2024-07-01T12:30:46.456+0300] {external_task.py:258} INFO - Poking for tasks ['end_task'] in dag fetch on 2024-06-30T00:00:00+00:00 ... 
[2024-07-01T12:31:46.473+0300] {external_task.py:258} INFO - Poking for tasks ['end_task'] in dag fetch on 2024-06-30T00:00:00+00:00 ... 
[2024-07-01T12:32:46.483+0300] {external_task.py:258} INFO - Poking for tasks ['end_task'] in dag fetch on 2024-06-30T00:00:00+00:00 ... 
[2024-07-01T12:33:42.780+0300] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-01T12:33:42.785+0300] {process_utils.py:132} INFO - Sending Signals.SIGTERM to group 2154. PIDs of all processes in the group: [2154]
[2024-07-01T12:33:42.786+0300] {process_utils.py:87} INFO - Sending the signal Signals.SIGTERM to group 2154
[2024-07-01T12:33:42.787+0300] {taskinstance.py:2611} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-07-01T12:33:42.788+0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-01T12:33:42.800+0300] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/user/rubix/internship_project/.venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/user/rubix/internship_project/.venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/user/rubix/internship_project/.venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/user/rubix/internship_project/.venv/lib/python3.10/site-packages/airflow/sensors/external_task.py", line 348, in execute
    super().execute(context)
  File "/home/user/rubix/internship_project/.venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/user/rubix/internship_project/.venv/lib/python3.10/site-packages/airflow/sensors/base.py", line 292, in execute
    time.sleep(self._get_next_poke_interval(started_at, run_duration, try_number))
  File "/home/user/rubix/internship_project/.venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2613, in signal_handler
    raise AirflowTaskTerminated("Task received SIGTERM signal")
airflow.exceptions.AirflowTaskTerminated: Task received SIGTERM signal
[2024-07-01T12:33:42.890+0300] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=Ingest_Data, task_id=wait_for_fetch_task, run_id=scheduled__2024-06-30T00:00:00+00:00, execution_date=20240630T000000, start_date=20240701T092546, end_date=20240701T093342
[2024-07-01T12:33:42.925+0300] {process_utils.py:80} INFO - Process psutil.Process(pid=2154, status='terminated', exitcode=2, started='12:25:46') (2154) terminated with exit code 2
